src/BribeInitiative.sol

```diff
@@ -99,28 +97,18 @@ contract BribeInitiative is IInitiative, IBribeInitiative {
             "BribeInitiative: invalid-prev-total-lqty-allocation-epoch"
         );
 
-        (uint88 totalLQTY, uint120 totalAverageTimestamp) = _decodeLQTYAllocation(totalLQTYAllocation.value);
```

Comment:
Can we remove now `utils/EncodingDecodingLib.sol`?

Reply:
Yes - done